opm/parser/eclipse/EclipseState/Schedule/TimeMap.hpp

```diff
@@ -38,7 +38,7 @@ namespace Opm {
     class TimeMap {
     public:
         TimeMap() = default;
-        explicit TimeMap(const Deck& deck);
+        explicit TimeMap(const Deck& deck, const std::pair<std::time_t, std::size_t>& restart = std::make_pair(0,0));
```

Comment:
This default value will generate conversions.  The `make_pair()` call will construct a `pair<int,int>` which will then be assigned/bound to the parameter type.  If you really want a pair of `std::size_t` you should say
```C++
restart = std::make_pair(std::size_t{0}, std::size_t{0})
```
On a separate note, you need to `#include <utility>` in order to reliably use `std::pair` as a parameter type here.

Reply:
I know - will update.

Reply:
Fixed